modules/ROOT/pages/connected-apps-developers.adoc

```diff
@@ -16,93 +16,183 @@ All scopes in the endpoint map directly to permissions available in Anypoint Pla
++
+.. *Grant types* +
+The grant types to be used with this app. +
+Note: Additional information may be requested based on the grant type you select. For more information on grant types, see xref:connected-apps-developers.adoc#grant-types-and-token-flow[Grant types and token flow].
++
+.. *Website URL* +
+Users can visit this URL to learn more about the app.
++
+.. *Redirect URIs* +
+Configure which URIs users should be directed to after authorization. Type the URL and click *Add* to add it to the list.
++
+.. *Scopes* +
+The level of permissions to be provided to the app. Click on the *Add Scopes* button to select the scopes. For more information, see xref:connected-apps-developers.adoc#endpoint-scopes[Endpoint Scopes].
++
+. Click *Save*
 
 [NOTE]
-Apps that use `client_credentials` grant type always have the `profile` scope assigned to them implicitly. The data returned by queries using the scope correspond to the user who created the connected app.
+====
+The connected app ownership gets transferred to the root organization owner
+when the user who created the connected app:
 
-For more information on scopes and permissions in the Connected Apps feature, see the scopes documentation page in Access Management:
+* is deleted
+* is removed from the root organization
+* loses administrative privileges over the organization
+
+By clicking on the app name in Access Management > *Connected Apps* > *Owned Apps*, organization administrators can edit the
+owner of the connected app and transfer the ownership to other administrators.
+====
+
+=== Modify the Client Secret for a Connected App
 
 include::partial$include-nav-connected-apps.adoc[]
 
-. Click the *Scopes Documentation* tab. +
-The *Scopes Documentation* page appears, displaying scopes, their respective permissions, and the products to which they belong. 
+. Click on the connected app from the list to enable Edit mode.
+. In the *Secret* field, enter a secret of your choice.
+. Click *Save*.
 
-You can filter and sort the contents of the *Scopes Documentation* page, and you can also click a scope name to view namespaces, methods, and actions.
+image::connected-apps-edit-mode.png[Edit mode for a connected app]
++
+[calloutlist]
+. The name and username of the owner of the connected app
+. The name of the connected app
+. The ID of the connected app
+. The client secret of the connected app
+
+[[application-allowlist-dev]]
+== Application Allowlist
 
-== OIDC Endpoints and Tokens
+Users with the Organization Administrator permission can enable an allowlist to control which apps can access their users' data.
+When your app is added to an allowlist, only the set of scopes associated with the app are reviewed and approved by the administrator.
+If you add new scopes to your app at a later date, you are unable to request those scopes from users until the admin re-approves your app.
+Existing authorizations and new authorizations using the already-allowlisted set of scopes continue to work.
```

Comment:
```suggestion
Existing authorizations and new authorizations using the previously allowlisted set of scopes continue to work.
```

Reply:
OR: Existing authorizations and new authorizations using the set scopes from the previous (original?) allowlist continue to work.